src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -388,6 +418,9 @@ export default {
 			const filters = convertQueryToFilters(query, allFacets, queryType, pageLimit);
 
 			await updateSearchState(apollo, filters, allFacets, queryType, loanSearchState);
+		},
+		addToBasket(payload) {
```

Comment:
I don't think we need this tracking. Definitely check locally, but the new loan card should already be adding the needed event. In general, we also need the loan and loan amount info for ATB events as well (but I think this should be covered already by the new loan card component).

Reply:
I think Lauren mentioned this as a requirement in the ticket, https://kiva.atlassian.net/browse/CORE-1174
- ensure tracking for add to basket 

Similar like is happening in every section for lending home tho

Reply:
That comment was referring to ensuring ATB events happened like expected, which they are inside the loan card. But having this extra event isn't bad either. I just think ATB events are generally less useful without the loan and amount info. Find to leave as is, but long-term we can probably remove this event and the similar ones from LH.

Reply:
Agreed, I think is for test metrics... thanks!

Reply:
I think the reports just check for experiment assignment.